scripts/module/keybinder.nut

```diff
@@ -1,36 +1,41 @@
-keys <- {
-  "a" : 0x41,
-    "b" : 0x42,
-    "c" : 0x43,
-    "d" : 0x44,
-    "e" : 0x45,
-    "f" : 0x46,
-    "g" : 0x47,
-    "h" : 0x48,
-    "i" : 0x49,
-    "j" : 0x4A,
-    "k" : 0x4B,
-    "l" : 0x4C
-    }; // FIXME: Complete the table
-
-
+dofile('./keytable.nut');
 binds <- array(100, null);
 
+function extractFirstWord( input ) {
+  local ret = {};
+  local firstSpace = input.find(" ");
+  if (!firstSpace) {
+    firstSpace = input.len()-1;
+  } else {
+    ret.rawset("second", input.slice(firstSpace+1));
+  }
+  ret.rawset("first", input.slice(0, firstSpace));
+  return ret;
```

Comment:
Why such a complicated datatype? Why not return `[before, after]`?

Reply:
Good suggestion, fixed!